webui-page/index.html

```diff
@@ -55,6 +55,27 @@
       <div id="playlist"></div>
     </div>
 
+    <div id="shortcuts-overlay" class="overlay">
+      <table class="shortcuts-table">
+        <tr><th><b class="fas fa-keyboard"></b></th><th>Action</th></tr>
+        <tr><td>Space</td><td>Play/Pause</td></tr>
+        <tr><td><b class="fas fa-arrow-right"></b></td><td>seek +10</td></tr>
+        <tr><td><b class="fas fa-arrow-left"></b></td><td>seek -10</td></tr>
+        <tr><td>PageDown</td><td>seek +3</td></tr>
+        <tr><td>PageUp</td><td>seek -3</td></tr>
+        <tr><td>f</td><td>toggle fullscreen</td></tr>
+        <tr><td>n</td><td>playlist next</td></tr>
+        <tr><td>p</td><td>playlist previous</td></tr>
+        <tr><td>[</td><td>decrease playback speed</td></tr>
+        <tr><td>]</td><td>increase playback speed</td></tr>
+        <tr><td>{</td><td>decrease playback speed more</td></tr>
+        <tr><td>}</td><td>increase playback speed more</td></tr>
+        <tr><td>Backspace</td><td>reset playback speed</td></tr>
+        <tr><td>Escape</td><td>hide current overlay</td></tr>
+        <tr><td>?</td><td>toggle keyboard shortcuts overlay</td></tr>
+      </table>
+    </div>
```

Comment:
If we'd have the shortcut configuration globally available in `webui.js`, we would be able to generate the shortcut table. The we'd have a single location for configuring shortcuts.